spec/javascript/containers/LocationsContainerSpec.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
I don't think this line is necessary because you've declared React as a global in your test helper